bigbluebutton-html5/private/config/settings.yml

```diff
@@ -29,6 +29,7 @@ public:
         chatAudioAlerts: false
         chatPushAlerts: false
         fallbackLocale: en
+        overrideLocale: 
```

Comment:
Is it best to keep it empty? Two lines below we have `undefined`, in the redis section we have `null`, ... What is the general recommendation?

Reply:
I changed it to `undefined`